v3.0/how-to/upgrade/from-previous-version.md

```diff
@@ -122,22 +147,26 @@ $ ansible-playbook local_prepare.yml
 
 > **Note:**
 >
-> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to  `PD service` under `systemd` in TiDB 3.0. Compared to earlier versions, rolling update operations on TiDB cluster components are slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
+> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to `PD service` under `systemd` in TiDB 3.0. If the current version is earlier than TiDB 3.0 versions, the rolling update of TiDB cluster components to the 3.0 versions is slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
```

Comment:
Please also update `dev/how-to/upgrade/from-previous-version.md` accordingly.

Reply:
`dev/how-to/upgrade/from-previous-version.md` only provide two links. There is no such content in it.

Reply:
Update that file according to changes [here](https://github.com/pingcap/docs-cn/pull/1574/files).